rmc/pipeline/regional_constraint.py

```diff
@@ -107,8 +114,7 @@ def main(args):
                 " significant break..."
             )
             if args.search_num == 1:
-                # TODO: Move existing HT to a different path so this can be remade
-                all_loci_chisq_ht_path = f"{SINGLE_BREAK_TEMP_PATH}/all_loci_chisq.ht"
+                all_loci_chisq_ht_path = f"{MODEL_PREFIX}/{CURRENT_GNOMAD_VERSION}/{args.freeze}/constraint_prep.ht"
```

Comment:
To fix in accordance with PR 306

Reply:
going to leave this open in the hopes that we'll merge 306 first (and then I can merge those changes into this PR)

Reply:
done!